### PR TITLE
Hotfix/project verification email

### PR DESCRIPTION
**Pull request by CarlosQ96**
To unblock Alireza mainly. But some work needs to be done on the dapp mailer app.

### CODE REVIEW

File: src/resolvers/projectVerificationFormResolver.ts [hunk 1]
```diff
@@ -49,6 +62,7 @@ export class ProjectVerificationFormResolver {
         throw new Error(errorMessages.PROJECT_VERIFICATION_FORM_NOT_FOUND);
       }
 
+      projectVerificationForm.emailConfirmationToken = null;
```
Comment (CarlosQ96) [inline src/resolvers/projectVerificationFormResolver.ts:65]: Once used, remove it from db

**Comment by CarlosQ96** [comment]
After merging the flow is:
1. Run the mutation to send the confirmation email. This will send the email to the recipient. Called: ```projectVerificationSendEmailConfirmation```
2. The email contains a url pointing to the dapp based on the env with the following format: staging.giveth.io/verification/[project-slug]/[token]

3. When the dapp opens this route, it should hit the mutation ```projectVerificationConfirmEmail```

Note: Mutations return and error or the verification form entity.

**Comment by CarlosQ96** [comment]
The mutations are:

1. Send confirmation Email:
```
mutation projectVerificationSendEmailConfirmation($projectVerificationFormId: Float!){
          projectVerificationSendEmailConfirmation(projectVerificationFormId: $projectVerificationFormId) {
            id
            isTermAndConditionsAccepted
            emailConfirmationToken
            emailConfirmationSent
            emailConfirmationSentAt
            emailConfirmedAt
            emailConfirmed
            projectRegistry {
              organizationDescription
              isNonProfitOrganization
              organizationCountry
              organizationWebsite
            }
            personalInfo {
              email
              walletAddress
              fullName
            }
            projectContacts {
              youtube
              instagram
              linkedin
              facebook
              instagram
              twitter
            }
            milestones {
              mission
              foundationDate
              achievedMilestones
              achievedMilestonesProof
            }
            managingFunds {
              description
              relatedAddresses {
                address
                networkId
                title
              }
            }
            user {
              id
              walletAddress
            }
            project {
              id
              slug
            }
            status
          }
        }
```
2. Confirm email:
```
mutation projectVerificationConfirmEmail($emailConfirmationToken: String!){
          projectVerificationConfirmEmail(emailConfirmationToken: $emailConfirmationToken) {
            id
            isTermAndConditionsAccepted
            emailConfirmationToken
            emailConfirmationSent
            emailConfirmationSentAt
            emailConfirmedAt
            emailConfirmed
            projectRegistry {
              organizationDescription
              isNonProfitOrganization
              organizationCountry
              organizationWebsite
            }
            personalInfo {
              email
              walletAddress
              fullName
            }
            projectContacts {
              youtube
              instagram
              linkedin
              facebook
              instagram
              twitter
            }
            milestones {
              mission
              foundationDate
              achievedMilestones
              achievedMilestonesProof
            }
            managingFunds {
              description
              relatedAddresses {
                address
                networkId
                title
              }
            }
            user {
              id
              walletAddress
            }
            project {
              id
              slug
            }
            status
          }
        }
```

**Comment by CarlosQ96** [comment]
Flow is:
Click button -> Frontend calls the send email mutation -> backend sends email -> user opens email and clicks link -> link opens dapp -> dapp call confirm email mutation -> mutation returns the entity succesfully with ```emailConfirmed = true``` OR return an error if expired or invalid

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @CarlosQ96 
LGTM

**Comment by MohammadPCh** [comment]
Thank you @CarlosQ96

**Pull request merged**